src/main/java/com/sourcegraph/webhook/Dispatcher.java

```diff
@@ -10,6 +10,7 @@
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
 import com.sourcegraph.webhook.registry.Webhook;
+import org.omg.CORBA.DynAnyPackage.Invalid;
```

Comment:
What on earth is this :D

Reply:
I think I was testing out IntelliJ type autocomplete to see if there were any useful classes to use and it auto-imported 😅It's removed from the latest commit.